Source/WebCore/rendering/FloatingObjects.cpp

```diff
@@ -112,6 +113,13 @@ LayoutSize FloatingObject::translationOffsetToAncestor() const
     return locationOffsetOfBorderBox() - renderer().locationOffset();
 }
 
+bool FloatingObject::isLowestPlacedFloatBottomInContainingBlock() const
```

Comment:
you may wanna put this behind #ifdef if only used in ASSERT

Reply:
Done